custom/icds_reports/tests/agg_tests/reports/test_bihar_api_demographics.py

```diff
@@ -40,6 +40,8 @@ def test_file_content(self):
              "registered_status": None, "rch_id": 1234,
              "mcts_id": 213, "phone_number": None,
              "date_death": None, "site_death": None,
-             "closed_on": None, "reason_closure": None},
+             "closed_on": None, "reason_closure": None, 'has_bank_account': None, 'age_marriage': None,
```

Comment:
can  we add some data in the fixture for atleast some of these fields to have non Nones here in the test cases?

Reply:
added